### PR TITLE
refactor(components): [time-picker] use useNamespace

### PR DESCRIPTION
**Pull request by gjfei**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue [hunk 1]
```diff
@@ -346,19 +350,20 @@ export default defineComponent({
...
         if (listRefsMap[type] && listRefsMap[type].$el) {
-          listRefsMap[type].$el.querySelector('.el-scrollbar__wrap').onscroll =
-            () => {
-              // TODO: scroll is emitted when set scrollTop programatically
-              // should find better solutions in the future!
-              handleScroll(type)
-            }
+          listRefsMap[type].$el.querySelector(
+            `.${ns.namespace.value}-scrollbar__wrap`
+          ).onscroll = () => {
```
Comment (JeremyWuuuuu) [inline packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue:357]:
```ts
getScrollbarElement(listRefsMap[type].$el).addEventListener(() => {
  handleScroll(type)
})
```

File: packages/components/time-picker/src/time-picker-com/panel-time-pick.vue [hunk 2]
```diff
@@ -77,13 +74,16 @@ export default defineComponent({
...
     const { t, lang } = useLocale()
     // data
     const selectionRange = ref([0, 2])
     const oldValue = useOldValue(props)
     // computed
     const transitionName = computed(() => {
-      return props.actualVisible === undefined ? 'el-zoom-in-top' : ''
+      return props.actualVisible === undefined
+        ? `${ns.namespace.value}-zoom-in-top`
+        : ''
```
Comment (JeremyWuuuuu) [inline packages/components/time-picker/src/time-picker-com/panel-time-pick.vue:86]: **Suggestion:**
```ts
import { isUndefined } from '@element-plus/utils'
      return isUndefined(props.actualVisible)
        ? `${ns.namespace.value}-zoom-in-top`
        : ''
```

**Review by jw-foss — COMMENTED** [review]
Left a couple of comments for you 🏓

**Comment by gjfei** [comment]
我不小心删除了我的分支

**Pull request closed without merging**